Packs/Genians/Integrations/Genians/Genians.py

```diff
@@ -0,0 +1,190 @@
+                tag_check = TAG_NAME
+                break
+
+        if tag_check == TAG_NAME:
+            hr = "IP : [{0}], [{1}] Tag assign success.".format(IP, TAG_NAME)
+            assign_tag = {
+                "nodeId" : nodeid,
+                "Name" : TAG_NAME
+            }
+            demisto.results({
+                'Type': entryTypes['note'],
+                'ContentsFormat': formats['json'],
+                'Contents': result2,
+                'ReadableContentsFormat': formats['text'],
+                'HumanReadable': hr,
+                'EntryContext': {
+                    "genian.nac.tag.(val.Tag == obj.Tag)": assign_tag
+                }
+            })
+        else:
+            raise Exception("IP : [{0}], [{1}] Tag assign fail.".format(IP, TAG_NAME))
+
+def unassign_ip_tag(nodeid:str, data):
+    URL = REQUEST_BASE_URL + "nodes/"+nodeid+"/tags?apiKey=" + APIKEY
+    result = http_request("DELETE", URL, body=data)
+    return result
+
+def unassign_ip_tag_command():
+    IP = demisto.getArg("ip")
+
+    result = get_ip_nodeid(IP)
+    nodeid = result[0]["nl_nodeid"]
+
+    if not nodeid:
+        demisto.results("IP not found. [{0}] is not exist in your network".format(IP))
+    else:
+        result2 = get_tag_list()
+
+        tag_check = "tag_not_exists"
+        for a in result2["result"]:
+            if a["NP_NAME"] == TAG_NAME:
+                tag_check = a["NP_IDX"]
+                break
+
+        if tag_check != "tag_not_exists":
+            if int(tag_check):
+                data = "[\""+str(tag_check)+"\"]"
+                result3 = unassign_ip_tag(nodeid, data)
+                if str(result3) == "[]":
+                    hr = "IP : [{0}], [{1}] Tag unassign success.".format(IP, TAG_NAME)
+                    unassign_tag = {
+                        "nodeId" : nodeid,
+                        "Name" : TAG_NAME
+                    }
+                    demisto.results({
+                        'Type': entryTypes['note'],
+                        'ContentsFormat': formats['json'],
+                        'Contents': result3,
+                        'ReadableContentsFormat': formats['text'],
+                        'HumanReadable': hr,
+                        'EntryContext': {
+                            "genian.nac.tag.(val.Tag == obj.Tag)": unassign_tag
+                        }
+                    })
+                else:
+                    raise Exception("IP : [{0}], [{1}] Tag unassign fail.".format(THREAT_IP, TAG_NAME))
+            else:
+                demisto.results("[{0}] Tag not found.".format(TAG_NAME))
+        else:
+            demisto.results("[{0}] Tag not found.".format(TAG_NAME))
+
+def main():
+    """Main execution block"""
+    try:
+
+        LOG("Command being called is {0}".format(demisto.command()))
+
+        if demisto.command() == "test-module":
+            demisto.results('ok')
+        elif demisto.command() == 'genian-nac-assign-ip-tag':

```

Comment:
Is it `genian` or `genians`?

Reply:
Thank you for answer.

Our company name is Genians, The product name is Genian NAC.
So, when running the integration module, it would be nice to name it genians-nac-xxxx.
But if you have to use genians-xxxx I will change it.